main/opensmtpd/smtpd.initd

```diff
@@ -2,23 +2,13 @@
 
 name="$SVCNAME"
 command="/usr/sbin/$SVCNAME"
-command_args="$SMTPD_OPTS"
+command_args="-F $SMTPD_OPTS"
+command_background=yes
+pidfile="/run/$SVCNAME.pid"
```

Comment:
This variable should be named `$RC_SVCNAME` (see [openrc-run(8)](http://manpages.org/openrc-run/8)). And the same applies for `name="$SVCNAME"` above.